pkg/channeld/connection.go

```diff
@@ -658,8 +596,10 @@ func (c *Connection) flush() {
 	if len > 0xff {

```

Comment:
if we consider tag[2] will always be  part of the packet length, this if statement is unnecessary.

Reply:
Removed the if statement.